jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile

```diff
@@ -2,13 +2,12 @@
  * The openstack-ardana Jenkins Pipeline
  */
 
-def ardana_lib = null
-
 pipeline {
   options {
     // skip the default checkout, because we want to use a custom path
     skipDefaultCheckout()
     timestamps()
+    timeout(time: 48, unit: 'HOURS')
```

Comment:
Do we really need a 48 hour timeout? Doesn't that seem rather long? :smile:

Reply:
Well, this is most like a last resort solution for jobs that get stuck and never finishes for some reason (mostly when something happens with the jenkins workers).

Besides that there are also some jobs that depending in which hosts they land on ECP it takes almost 48h to finish :smiley: see (https://ci.suse.de/blue/organizations/jenkins/cloud-ardana9-job-gate-entry-scale-kvm-update-x86_64/detail/cloud-ardana9-job-gate-entry-scale-kvm-update-x86_64/98/pipeline)

Reply:
Thanks :smile: